### PR TITLE
feat: run transform on import

### PR DESCRIPTION
**Pull request by Cooryd**

### CODE REVIEW

File: src/config/default.ts [hunk 1]
```diff
@@ -15,5 +15,7 @@ export const defaultConfig: Config = {
   dbSSLValidate: false,
   dbName: "editor",
   editorS3Bucket: "editor",
-  srcS3Bucket: "kryia"
+  srcS3Bucket: "kryia",
```
Comment (NuclearRedeye) [inline src/config/default.ts:18]: Something for a future follow up, but we should probably remove eLife specific terms from the defaults at some point.

File: src/listeners/import-handler.ts [hunk 2]
```diff
@@ -3,8 +3,9 @@ import { Db } from "mongodb";
 import decompress from "decompress";
 import FileType from "file-type";
 import { AssetService } from '../services/asset';
+import { TransformService } from '../services/transform';
 
-export default function importHandler(assetService: AssetService, db: Db) {
+export default function importHandler(assetService: AssetService, transformService: TransformService, db: Db, transformEnabled?: boolean) {
```
Comment (NuclearRedeye) [inline src/listeners/import-handler.ts:8]: I've always preferred default values over optional parameter? e.g. `transformEnabled: boolean = false`. Up to you if that's something that you want to change or not.

File: src/services/config-manager.ts [hunk 3]
```diff
@@ -3,17 +3,17 @@ import { ConfigManager } from '../types/config-manager';
 
 // Encapsulates a Configuration
 export class ConfigManagerInstance implements ConfigManager {
-  private config: Map<string, string>;
+  private config: Map<string, any>;
```
Comment (NuclearRedeye) [inline src/services/config-manager.ts:6]: Interesting, but perhaps this could be `Map<string, string | boolean>` instead to make it a more explicit?

**Review by NuclearRedeye — APPROVED** [review]
Some minor comments, but overall the changes look good.

File: src/config/default.ts (hunk 1 above)
Comment (Cooryd) [reply src/config/default.ts:18]: https://app.zenhub.com/workspaces/libero-editor-5d56a6cfaa13c900012ad75b/issues/libero/editor/801 should remove this value.

File: src/services/config-manager.ts (hunk 3 above)
Comment (Cooryd) [reply src/services/config-manager.ts]: Which has caused compiler errors 😄 I'll go back to the TS docs on this one.

**Pull request merged**